### ISSUE TITLE
Comment not added to context on reset

### ISSUE THREAD

**Issue by asztalosdani** [pyblish/pyblish-qml#280]
### Description
The comment is not added to the context after hitting reset on GUI
### Expected result
The comment should be added to the context every time.
### How to reproduce
```python
import pyblish.api

class CommentValidator(pyblish.api.ContextPlugin):
    order = pyblish.api.ValidatorOrder
    
    def process(self, context):
        assert "comment" in context.data
        
pyblish.api.register_plugin(CommentValidator)
```

0. Open pyblish-qml GUI
1. Provide a comment
2. Hit Validate -> it passes
3. Hit Reset
4. Validate -> it fails but the comment is there

**Comment by BigRoy** [comment]
@asztalosdani Thanks - I was actually about to report this too since I had noticed it a day or two again as well. 

So thanks a lot for taking the time to generate a reproducible case. In my case it actually passed, but it's because the resulting data ended up being empty.

```python
import pyblish.api

class CommentValidator(pyblish.api.ContextPlugin):
    order = pyblish.api.ValidatorOrder
    
    def process(self, context):
        assert context.data["comment"], "Must have comment content!"
        
pyblish.api.register_plugin(CommentValidator)
```

For me the above was reproducible. It seems that whenever, *after reset* the comment text is not changed by the artist it is not placed in the context. The comment is still visible within the comment field, it's just **not** applied to the Context correctly.

**Comment by mottosso** [comment]
Can confirm that I have been experiencing oddities, the culprit is most likely a logical/human error around [these parts](https://github.com/pyblish/pyblish-qml/blob/927ce409f9e9219cb3fa244d52a48ecb71c841e6/pyblish_qml/control.py#L742).

**Comment by davidlatwe** [comment]
Oh, just found this issue, fixed with #299 :)
Closing now.
